1.setup.sh

```diff
@@ -64,16 +65,30 @@ idem_cmd git
 idem "command -v curl" "sudo apt install curl"
 
 function init_aws () {
+  # install CLI
   mkdir ~/awstmp
   pushd ~/awstmp
   curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
   unzip ./awscliv2.zip
   sudo ./aws/install
   popd
   rm -rf ~/awstmp
+  # defaults for CLI auth
+  mkdir ~/.aws
+  echo "[profile default]" > ~/.aws/config
+  echo "sso_start_url = $SSO_START_URL" > ~/.aws/config
+  echo "sso_region = us-west-2" > ~/.aws/config
+  # trigger initial log in
   echo "This script will start the SSO flow for AWS, opening a browser."
-  echo "Please select your default role for the 'sand' account."
-  aws configure sso --profile sand
+  echo "Please..."
+  echo "1. accept defaults as given"
+  echo "2. select the 'sand' account"
+  echo "3. choose your default role for the 'sand' account"
+  echo "4. call this profile 'sand' so that documentation and scripts can use it"
+  aws configure sso
+  SAND_ACCOUNT_ID = $(aws sts get-caller-identity --query Account --output text --profile sand)
+  aws configure set role_arn arn:aws:iam::$SAND_ACCOUNT_ID:role/cicd --profile sand-cicd
+  aws configure set source_profile sand --profile sand-cicd
```

Comment:
Running these three preceeding lines (not `aws configure sso`) will set you up for running cicd scripts under the same identity that GitHub will use.  (this will alter `~/.aws/config`)